apps/requestron/lib/milestone.js

```diff
@@ -5,7 +5,7 @@ const axios = require('axios')
 
 const instance = axios.create({
   baseURL: 'https://api.github.com/',
-  timeout: 1000,
+  timeout: 10000,
```

Comment:
that's a very long timeout ~ 3 hours! what's the case for this wait time? Maybe instead setup a retry?

Reply:
thats in milliseconds :)